File: tabs-api/app/constants/genre.py
from enum import Enum

class Genre(str, Enum):
    """
    Constants for the different music genres
    """
    CLASSICAL = "classical"
    COUNTRY = "country"
    FOLK = "folk"
    INDIE = "indie"
    METAL = "metal"
    POP = "pop"
    ROCK = "rock"
    CHRISTMAS = "christmas"

File: client/src/constants/genre.ts
export const Genres = {
    ROCK: "rock",
    JAZZ: "jazz",
    INDIE: "indie",
    INDIE_ROCK: "indie rock",
    POP: "pop",
    METAL: "metal",
    FOLK: "folk"
} as const;

export type Genre = (typeof Genres)[keyof typeof Genres];
